### PR TITLE
Java APIs Talk updated

### PR DESCRIPTION
**Pull request by hendrikebbers**
Java APIs Talk updated

### CODE REVIEW

File: _data/talks.yml [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+    Based on an overview of this topics and some samples the talk will answer questions like: 
+
+    - How can I create my own annotations? 
+    
+    - How can I create a plugin structure without using frameworks like OSGI? 
+    
+    - What’s the best way to handle NullPointerExceptions? 
+    
+    - How can I write concurrent code that is still maintainable? 
+  lectures: ['Dev Talks Cluj 2018', 'w-jax 2018']
```
Comment (madmas) [inline _data/talks.yml]:
```suggestion
  lectures: ['Dev Talks Cluj 2018', 'W-JAX 2018']
```

File: _data/talks.yml [hunk 2]
```diff
@@ -1,4 +1,25 @@
 # Data file that describes talks
+- name: 'Java APIs - The missing manual'
+  author: 'hendrik'
+  summary: >
+    This isn’t a talk about microservices, NO-SQL, Container solutions or hip new frameworks.
```
Comment (madmas) [inline _data/talks.yml]:
```suggestion
    This isn’t a talk about microservices, NoSQL, container solutions or hip new frameworks. 
```

File: _data/talks.yml [hunk 3]
```diff
@@ -1,4 +1,25 @@
 # Data file that describes talks
+- name: 'Java APIs - The missing manual'
+  author: 'hendrik'
+  summary: >
+    This isn’t a talk about microservices, NO-SQL, Container solutions or hip new frameworks. 
+    This talk will show some of the standard Java APIs that are part of Java since version 5, 6, 7 or 8.
+    All this features are very helpful to create maintainable and future-proof applications, regardless
```
Comment (madmas) [inline _data/talks.yml]:
```suggestion
    All those features are very helpful to create maintainable and future-proof applications, regardless
```

File: _data/talks.yml [hunk 4]
```diff
@@ -56,23 +77,3 @@
   youtube-link: 'https://www.youtube.com/watch?v=vDQQ8axRC8I'
   featuredImage: mittie-talk
 
-- name: 'Java APIs - The missing manual'
```
Comment (madmas) [inline _data/talks.yml]: Is it the plan to move talk section to the beginning of the file instead of updating them in place?

Comment (hendrikebbers) [reply _data/talks.yml]: Currently the first on is on top of the rendered page. I think we should add a date or something to the data model and than sort it automatically ;) https://github.com/karakun/karakun.github.io/issues/97

**Pull request merged**